medusa/__main__.py

```diff
@@ -473,12 +473,14 @@ def initialize(self, console_logging=True):
                     app.GIT_REMOTE_URL = app.APPLICATION_URL
 
             # current commit hash
-            app.CUR_COMMIT_HASH = check_setting_str(app.CFG, 'General', 'cur_commit_hash', '')
+            commit_hash_env = os.environ.get('MEDUSA_COMMIT_HASH', None)  # This is for our Docker container
+            app.CUR_COMMIT_HASH = check_setting_str(app.CFG, 'General', 'cur_commit_hash', commit_hash_env or '')
```

Comment:
I just realized these changes might not work as expected.
Need to look into it more, but when updating the container after pulling a new image, the old hash/branch will remain...